docs/modules/cluster-performance/pages/performance-tuning.adoc

```diff
@@ -147,7 +147,7 @@ We xref:ROOT:production-checklist.adoc#jvm-recommendations[recommend] enabling
 GC logs to allow troubleshooting if performance problems occur. To enable GC
 logging, use the following JVM arguments:
 
-*Java 11 (and above)*
+*Java 17 (and above)*
```

Comment:
Nit - not sure what value this adds? There's no non-Java 17 option for the user, or in the documentation

Also we said "Java 17+" earlier. Is it "+" or "above"?